scripts/tap-generate-docs/src/tap_generate_docs.clj

```diff
@@ -51,14 +51,13 @@
             other-properties (let [other-properties (if (string? item-type)
                                                       [item-type]
                                                       (filter (partial not= "object") item-type))]
-                               (if ((set other-properties) "array")
-                                 (throw (ex-info "Currently cannot handle a type with object _and_ array present"
-                                                 {:property property}))
-                                 other-properties))
-            converted-other-properties [(convert-multiary-type schema ["value"
-                                                                       (assoc property-json-schema-partial
-                                                                              "type"
-                                                                              other-properties)])]
+                               other-properties)
+            converted-other-properties (if (= '("null") other-properties)
```

Comment:
Why not `(if (= ["null"] other-properties))` here?

Reply:
I believe we observed that the value at this point came through as a list, but looking at it now, it seems like it could be a vector. I'm still kind of fuzzy at the distinction between the two with regard to equality.

Reply:
Nm, we changed it to a list in a later commit. See other comment.